t/plugin/api-breaker.t

```diff
@@ -384,41 +390,89 @@ passed
 
 
 
-=== TEST 15: max_breaker_seconds = 20
+=== TEST 12: default value
+--- request
+GET /api_breaker?code=500
+--- error_code: 500
+--- no_error_log
+[error]
+
+
+
+=== TEST 13: trigger default value of unhealthy.http_statuses breaker
+--- request eval
+["GET /api_breaker?code=200", "GET /api_breaker?code=500", "GET /api_breaker?code=503", "GET /api_breaker?code=500", "GET /api_breaker?code=500", "GET /api_breaker?code=500"]
+--- error_code eval
+[200, 500, 503, 500, 500, 502]
+--- no_error_log
+[error]
+
+
+
+=== TEST 14: trigger timeout 2 second
+--- config
+    location /sleep1 {
```

Comment:
do we really need it? please confirm!

You are writing test cases, if the case is incorrect, it means your work is meaningless.